pkg/util/namespace/namespace_test.go

```diff
@@ -7,62 +7,30 @@ import (
 	"testing"
 )
 
-func TestIsOpenShift(t *testing.T) {
+func TestFilteredOpenShiftNamespace(t *testing.T) {
 	for _, tt := range []struct {
 		namespace string
 		want      bool
 	}{
 		{
-			want: true,
-		},
-		{
-			namespace: "openshift-ns",
-			want:      true,
-		},
```

Comment:
Instead of removing test cases I would update them to reflect new filtering. So this one becomes `want: false`.